config/deploy/docker/lib/deployer.rb

```diff
@@ -253,7 +254,9 @@ def _add_latest_tag!
       image_tag: image_tag_latest,
       image_manifest: manifest
     }
-    ecr.put_image(putparams)
+    response = ecr.put_image(putparams)
+    File.open("./tmp/latest-tag-log--#{image_tag}--#{image_tag_latest}--#{Time.now.strftime('%m-%d--%H:%M:%S')}", 'w') { |file| file.write(response.to_h.inspect) }
+    puts ">> Latest tag '#{image_tag_latest}' is now even with '#{image_tag}'"
```

Comment:
Can this just be done with `Rails.logger.info`? Seems odd to open a file and write to it, but maybe that's ok.

Reply:
If you're just dropping the file as a log for archival purposes, then maybe just:
```
logfile = File.join('tmp',  "latest-tag-log--#{image_tag}--#{image_tag_latest}--#{Time.current.to_s(:db)}")
File.write(logfile, response.to_h.inspect)
```
(We like File.join to support various host operating systems, and we prefer Time.current over Time.now since it includes timezone information.)

Reply:
I should have clarified that this increased logging doesn't need to be permanent. The file writing part especially I intend to remove after we nail down the latest tag issue. The idea was just to get a quick, easy-to-access record of the response data from that AWS CLI image push, in case the Friday deployment doesn't do the latest tags correctly. I suppose there's no reason not to log it to the regular log though, so I'll switch to doing that (sometimes I just prefer to log directly to a file because I always know where that will end up, no matter what project I'm in).

Reply:
Change made.